example_configs/harbor.md

```diff
@@ -0,0 +1,21 @@
+You can pass environment variables into ``harbor-core`` for auth configuration as documented [here](https://github.com/goharbor/website/blob/release-2.10.0/docs/install-config/configure-system-settings-cli.md#harbor-configuration-items).
```

Comment:
Can you add a (very) quick word on what is Harbor?

Reply:
Done.